src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -557,6 +557,17 @@ public static Path getPath(String uriString) {
         }
     }
 
+    /**
+     * Gets the absolute Path name with the URI marker, handling the special case of the default file system by removing
+     * the file:// prefix.
+     *
+     * @param path path to get the absolute name.
+     * @return a String with the absolute name.
+     */
+    public static String getAbsolutePathName(final Path path) {
```

Comment:
The name of this method doesn't really tell you what it's doing -- can you rename to `getAbsolutePathWithoutFileProtocol()`?

Reply:
Done.